mysql-compatibility.md

```diff
@@ -39,6 +39,7 @@ However, some features of MySQL are not supported. This could be because there i
 + `CHECK TABLE` syntax [#4673](https://github.com/pingcap/tidb/issues/4673)
 + `CHECKSUM TABLE` syntax [#1895](https://github.com/pingcap/tidb/issues/1895)
 + `GET_LOCK` and `RELEASE_LOCK` functions [#14994](https://github.com/pingcap/tidb/issues/14994)
++ `HANDLER` commands
```

Comment:
Can we refer to it as [`HANDLER` statement](https://dev.mysql.com/doc/refman/8.0/en/handler.html)?

We have some other statements not supported if you can document at the same time:
https://github.com/pingcap/docs/issues/7981
https://github.com/pingcap/docs/issues/7980

Reply:
> Can we refer to it as [`HANDLER` statement](https://dev.mysql.com/doc/refman/8.0/en/handler.html)?
> 
> We have some other statements not supported if you can document at the same time: #7981 #7980

OK. I will document them as well. But do you know whether they have any associated issues? It seems other unsupported features are linked to issues.

Reply:
I think it's fine to document without issues. We likely do have issues in pingcap/tidb if you want to search for them though.